tests/hmm/plotting.py

```diff
@@ -773,11 +827,14 @@ def PrintDatacard(categories, event_counts, filenames, ofname):
 
                 histos = {s: res[s][analysis][var] for s in mc_samples + ["data"]}
                 print(era, analysis, var)
+                combined_signal_samples= ["ggh_amcPS", "vbf", "vh", "tth"]
```

Comment:
Same, here, could you load these from pars.py? Just to be able to change and review all the configuration options in one place.

Reply:
done